ezsnmp/session.py

```diff
@@ -288,11 +291,20 @@ def format_hostname(hostname):
             return self.hostname
 
     @property
-    def timeout_microseconds(self):
+    def timeout_microseconds(self) -> int:
         # Calculate our timeout in microseconds
         return int(self.timeout * 1000000)
 
-    def get(self, oids):
+    @overload
```

Comment:
why overload the `get` function ?

- [ ] ?

Reply:
if only one oid is input, return value is `SNMPVariable`, if `oids`(list) are provided, `list[SNMPVariable]` will return.
overload is only for IDE with correct type hint when user has different input.

Reply:
you can check the type hint from IDE for test file to see if it's meet the expect